README.md

```diff
@@ -1,55 +1,36 @@
 # nixpkgs-graph-explorer
 
-> This project is currently in a prototype state.
-
 ## Overview
 
-This project aims to make the software dependency graph data from `nixpkgs` more
-easily accessible for analytics and general exploration use cases. It does this
-by extracting the dependency graph for a single `nixpkgs` revision and loading
-it to a graph database compatible with the
-[Gremlin](https://tinkerpop.apache.org/gremlin.html) graph traversal language
-(sqlg + Postgres).
-
-The repository is currently split into a few parts, each with its own
-documentation:
-
-* `api` The REST API for serving the graph
-* `etl` Scripts for extracting the nixpkgs graph and loading it to Gremlin
-  Server
-* `config` Various configuration
-* `docker` Docker image definitions, etc.
-* `web` Web app to interact with the API
+This project aims to make the software dependency graph data from `nixpkgs` more easily accessible for analytics and general exploration use cases.
 
 ## Running locally
 
-To spin up the application locally you can use the Docker Compose stack defined
-in [docker-compose.yaml](docker-compose.yaml). This will create a Postgres
-database, a Gremlin Server which proxies the Postgres database, and the latest
-version of the `nixpkgs-graph-explorer` API.
+```
+docker-compose --profile all up
+```
+
+The web UI is available at [http://localhost:9091](http://localhost:9091).
 
-We currently divide the services in `docker-compose.yaml` into several profiles:
+The graph data is not automatically loaded.
+See instructions in [api](./api) to load data.
 
-| profile  | services                              |
-|----------|---------------------------------------|
-| `db`       | Gremlin Server, Postgres              |
-| `backend`  | Gremlin Server, Postgres, API         |
-| `frontend` | Web UI                                |
-| `all`      | Gremlin Server, Postgres, API, Web UI |
+<!-- TODO add instructions to load data here -->
 
+## Development
 
-First, you'll need to build the images:
+We use [Nix](https://nixos.org/) to manage dependencies, so you need
+[Nix: the package manager](https://nixos.org/download.html) installed to develop
+nixpkgs-graph-explorer. Also, because we use advanced features of Nix (such as flakes), we
+recommend you turn them ON by default with an alias (in `.bashrc` for example):
 
 ```
-docker-compose --profile all build
+alias nix='nix --extra-experimental-features nix-command --extra-experimental-features flakes'
```

Comment:
~~~suggestion
We use [Nix](https://nixos.org/) to manage development dependencies, so you need Nix [installed](https://nixos.org/download.html) to develop nixpkgs-graph-explorer.
Also, because we use advanced features of Nix (such as flakes), we recommend that you turn them ON by default.

To do so, add to `~/.config/nix/nix.conf`:

```
experimental-features = nix-command flakes
```
~~~